Analysis/include/EventInfo.h

```diff
@@ -199,7 +208,10 @@ class EventInfoBase {
 
         const JetCollection& all_jets = GetJets();
         JetCollection selected_jets;
-        for(const JetCandidate& jet : all_jets) {
+        const std::set<size_t> bjet_indexes = GetSelectedBjetIndices();
```

Comment:
bjet_indexes should be passed as an argument: not always we need to exclude bjets.